src/components/CommentSection/CommentSectionItem.tsx

```diff
@@ -28,6 +28,7 @@ export function CommentSectionItem({ comment, modelId }: Props) {
     { commentId: comment.id },
     { initialData: comment.reactions }
   );
+  const { data: model } = trpc.model.getById.useQuery({ id: comment.modelId });
```

Comment:
Did you confirm that this doesn't lead to additional queries or whatever as the page is loading?

Reply:
Since we are prefetching the model data in `getServerSideProps`, there won't be another query to the API. That's the beauty of react-query 🎉